lib/modules/collection/collection_providers.dart

```diff
@@ -46,6 +46,11 @@ final entriesProvider = Provider.autoDispose.family(
             break;
           }
         }
+
+        if (entry.notes != null && entry.notes!.toLowerCase().contains(search)) {
```

Comment:
Maybe this should be wrapped in `if (!contains) ...` to skip entries that already pass? I don't know how much I should worry about this, but we are in a tight loop after all and a possible future feature is an "All Lists" view, which means all entries will have to pass through filtering.